### PR TITLE
FAQ: update uninstall method

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: share/doc/homebrew/FAQ.md [hunk 1]
```diff
@@ -44,14 +44,14 @@ to see what would be cleaned up:
...
 ### How do I uninstall Homebrew?
-If you installed to `/usr/local` then you can use the script in [this
-gist](https://gist.github.com/1173223) to uninstall — it will only
-remove Homebrew and the stuff Homebrew installed leaving anything else
-in `/usr/local` alone.
-
-Provided you haven’t put anything else in Homebrew’s prefix
-(`brew --prefix`), you can generally just `rm -rf` that directory. This
-is because Homebrew won’t touch files outside its prefix.
+To uninstall Homebrew, paste below command at a Terminal prompt.
```
Comment (mikemcquaid) [inline share/doc/homebrew/FAQ.md]: `To uninstall Homebrew, paste the command below in a terminal prompt`

File: share/doc/homebrew/FAQ.md [hunk 2]
```diff
@@ -44,14 +44,14 @@ to see what would be cleaned up:
...
+
+```bash
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall)"
+```
+
+There are more uninstall options. Download [uninstall script](https://raw.githubusercontent.com/Homebrew/install/master/uninstall)
+to local and run `./uninstall --help` to check them.
```
Comment (mikemcquaid) [inline share/doc/homebrew/FAQ.md]: `Download the uninstall script and run `./uninstall --help` to view more uninstall options.`

**Comment by xu-cheng** [comment]
Updated. Sorry about my poor English.

**Comment by mikemcquaid** [comment]
Don't be sorry, it's better than my Chinese :)

**Comment by mikemcquaid** [comment]
:+1: :shipit:

**Pull request closed without merging**